api/controllers/TelevisionController.js

```diff
@@ -0,0 +1,328 @@
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+     /**
+     * @api {post} /television/volume Set Volume 
+     * @apiName televisionSetVolume
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * @apiParam {integer} volume Volume level
+     */
+    setVolume: function(req, res, next){
+        gladys.television.setVolume(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/stop Stop 
+     * @apiName televisionStop
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    stop: function(req, res, next){
+        gladys.television.stop(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volumeUp Volume Up 
+     * @apiName televisionVolumeUp
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeUp: function(req, res, next){
+        gladys.television.volumeUp(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volumeDown Volume Down 
+     * @apiName televisionVolumeDown
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeDown: function(req, res, next){
+        gladys.television.volumeDown(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/pressKey Press Key 
+     * @apiName televisionPressKey
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    pressKey: function(req, res, next){
+        gladys.television.pressKey(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/getSources Get Sources
+     * @apiName televisionGetSources
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     * @apiSuccess {array} All television source
+     */
+    getSources: function(req, res, next){
+        gladys.television.getSources(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/openSources Open Sources
+     * @apiName televisionOpenSources
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    openSources: function(req, res, next){
+        gladys.television.openSources(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/openMenu Open Menu
```

Comment:
Same here!

Reply:
Fix this on commit c62f1694228a420b53efebcb015d877208a3c3d3